closed/OpenJ9.gmk

```diff
@@ -479,6 +479,11 @@ AllJclSource   = $(call recur_wildcard,$(OPENJ9_TOPDIR)/jcl/src,*.java)
 
 JPP_DEST := $(SUPPORT_OUTPUTDIR)/j9jcl_sources
 
+CONFIG_VERSION := JAVA$(VERSION_FEATURE)
+ifeq (true,$(OPENJ9_ENABLE_INLINE_TYPES))
+  CONFIG_VERSION := INLINE-TYPES
+endif
```

Comment:
This needs to be replace by code that influences the `-tag:define` option below - like:
```
JPP_TAGS := PLATFORM-$(OPENJ9_PLATFORM_CODE)
ifeq (true,$(OPENJ9_ENABLE_INLINE_TYPES))
  JPP_TAGS := $(JPP_TAGS);INLINE-TYPES
endif
```